sections/bt-header.liquid

```diff
@@ -35,11 +35,27 @@
 
     {% if section.settings.notification_bar_text != blank %}
       {% if section.settings.notification_bar_link != blank %}
-        <a href="{{section.settings.notification_bar_link}}" class='table--cell-center'>
-          <span class="header-message-text">{{ section.settings.notification_bar_text | escape }}</span>
+        <a href="{{section.settings.notification_bar_link}}">
+        <div class="notification-header">
+        {% if section.settings.notification_bar_enable_image != blank %}
+          <div class="notification-image-wrapper"><img
+                    class="javascript-load-image"
+                    src="{{section.settings.notification_bar_image | img_url: '150x' }}"
+          ></div>
+          {% endif %}
+          <div class="notification-text" class="header-message-text">{{ section.settings.notification_bar_text }}</div>
+          </div>
         </a>
       {% else %}
-        <span class="header-message-text table--cell-center text-center">{{ section.settings.notification_bar_text | escape }}</span>
+        <div class="notification-header">
+        {% if section.settings.notification_bar_enable_image != blank %}
+          <div class="notification-image-wrapper"><img
+                    class="javascript-load-image"
+                    src="{{section.settings.notification_bar_image | img_url: '150x' }}"
+          ></div>
+          {% endif %}
+          <div class="notification-text" class="header-message-text">{{ section.settings.notification_bar_text }}</div>

```

Comment:
Rather than putting a class on the div can we just slap a `<p>` around this and let the site typography take over?